watertap/unit_models/electrodialysis_1D.py

```diff
@@ -103,11 +117,56 @@ class Electrodialysis1DData(UnitModelBlockData):
     CONFIG.declare(
         "operation_mode",
         ConfigValue(
-            default="Constant_Current",
-            domain=In(["Constant_Current", "Constant_Voltage"]),
+            default=ElectricalOperationMode.Constant_Voltage,
+            domain=In(ElectricalOperationMode),
             description="The electrical operation mode. To be selected between Constant Current and Constant Voltage",
         ),
     )
+    CONFIG.declare(
+        "limiting_current_density_method",
+        ConfigValue(
+            default=LimitingCurrentDensityMethod.Constant_input,
```

Comment:
After a second thought, I rename it to be `InitialValue` because the limiting current density isn't exactly constant over the cell length. Note is given as doc when declaring the config.